<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>556</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Отрезки</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="mainview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>511</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="what">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="circle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Окружность </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ellips">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Эллипс </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="other">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>321</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="a">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="middle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Средней точки</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="bground_color">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="lib">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Библиотечный</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dia">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="angle_step">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Количество </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="rad">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="draw_once">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отобразить окружность </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="canon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Каноническое уравнение </string>
     </property>
    </widget>
    <widget class="QPushButton" name="draw_centr">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Концентр</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="param">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Параметрическое уравнение </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="b">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>300</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clean_all">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Очистить холст</string>
     </property>
    </widget>
    <widget class="QLabel" name="diametr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Шаг </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="brez">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Брезенхем</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="centr_x">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="count">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>400</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>300</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="centr_y">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать цвет линии</string>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Радиус </string>
     </property>
    </widget>
    <widget class="QLabel" name="end_point">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Центр</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="line_color">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_bground">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать цвет фона</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>460</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Для окружности:</string>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Для эллипса:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
